stable/redis-ha/templates/redis-server-statefulset.yaml

```diff
@@ -1,5 +1,20 @@
-apiVersion: extensions/v1beta1
-kind: Deployment
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ template "redis-ha.fullname" . }}-server-svc
+  labels:
+    chart: {{ template "chartref" . }}
```

Comment:
Why not use the "labels.standard" here as well? As you do that for the Sentinel server, I'd expect it here as well.

Reply:
Done